src/SDK.ts

```diff
@@ -231,7 +237,9 @@ export default class SDK implements ISDK {
     const headers: StringMap = Constants.defaultHeaders;
     let requestPath = `/${OmnichannelEndpoints.LiveChatGetChatTokenPath}/${this.omnichannelConfiguration.orgId}/${this.omnichannelConfiguration.widgetId}/${requestId}`;
 
-    if (this.liveChatVersion === LiveChatVersion.V2 || (currentLiveChatVersion && currentLiveChatVersion === LiveChatVersion.V2)) {
+    if (this.liveChatVersion === LiveChatVersion.V3 || (currentLiveChatVersion && currentLiveChatVersion === LiveChatVersion.V3)) {
+      requestPath = `/${OmnichannelEndpoints.LiveChatv3GetChatTokenPath}/${this.omnichannelConfiguration.orgId}/${this.omnichannelConfiguration.widgetId}/${requestId}`;
+    } else if (this.liveChatVersion === LiveChatVersion.V2 || (currentLiveChatVersion && currentLiveChatVersion === LiveChatVersion.V2)) {
       requestPath = `/${OmnichannelEndpoints.LiveChatv2GetChatTokenPath}/${this.omnichannelConfiguration.orgId}/${this.omnichannelConfiguration.widgetId}/${requestId}`;
```

Comment:
this could be simplified by defining the token endpoint separately , reducing the need to duplicate code,  an idea like -> 

tokenPath = this.liveChatVersion === LiveChatVersion.V3? OmnichannelEndpoints.LiveChatv3GetChatTokenPath : OmnichannelEndpoints.LiveChatv2GetChatTokenPath ;

requestPath = `/${tokenPath}/${this.omnichannelConfiguration.orgId}/${this.omnichannelConfiguration.widgetId}/${requestId}`;

Reply:
The code got refactored. 

On SDK init, it has a default `liveChatVersion` (1). 

Usually, a get chat config call would fetch the `liveChatVersion` from the service, overwrite that value, then the API calls version will be used accordingly. 

We have situation where chat config is cached, so the `liveChatVersion` will always be `1`. So, we need to explicitly pass `currentLiveChatVersion` from the cache to the SDK to know which API call to use. 

So, `currentLiveChatVersion` should always take precedence on than `liveChatVersion`